src/Symfony/Component/Security/Core/Authentication/Token/NullToken.php

```diff
@@ -42,6 +42,13 @@ public function setUser($user)
     }
 
     public function getUsername()
+    {
+        trigger_deprecation('symfony/security-core', '5.3', 'Method "%s()" is deprecated and will be removed in 6.0, use getUserIdentifier() instead.', __METHOD__);
```

Comment:
IIRC we removed the `"and will be removed in X.0"` from deprecation notices a while ago